proxygen/lib/http/codec/compress/experimental/qcram/QCRAMEncoder.cpp

```diff
@@ -0,0 +1,281 @@
+/*
+ *  Copyright (c) 2017, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+#include <proxygen/lib/http/codec/compress/experimental/qcram/QCRAMEncoder.h>
+
+#include <proxygen/lib/http/codec/compress/experimental/simulator/QCRAMNewScheme.h>
```

Comment:
Should file 'proxygen/lib/http/codec/compress/experimental/simulator/QCRAMNewScheme.h' be part of this diff?

Reply:
Yes, sorry about that.  I've updated the branch.